### PR TITLE
PS-7254 Document Data Masking Plugin in 5.7 (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/index.rst
	new file:   source/security/data-masking.rst

### CODE REVIEW

File: doc/source/security/data-masking.rst [hunk 1]
```diff
@@ -0,0 +1,191 @@
+.. _data-masking:
+
+==================================================================
+Data Masking
+==================================================================
+
+This feature is **Experimental** quality.
+
+This feature was implemented in |Percona Server| version :rn:`5.7.31-34`.
```
Comment (george-lorch) [inline doc/source/security/data-masking.rst]: This will not be in 5.7.31-34 but next series 5.7.32 or maybe an interim release of 5.7.31-35

**Review by george-lorch — APPROVED** [review]
Looks good but please do not merge until https://github.com/percona/percona-server/pull/3993 is merged

**Review by kathy-williamson — CHANGES_REQUESTED** [review]
The following command installs the plugin and register the UDF functions:  S/B commands install (there are multiple commands)

File: doc/source/security/data-masking.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/security/data-masking.rst]: Added a placeholder.

**Comment by patrickbirch** [comment]
Fixed.

**Comment by patrickbirch** [comment]
Updated and awaiting comments.

**Review by kathy-williamson — APPROVED** [review]

**Pull request merged**